cv32e40s/tests/programs/custom/pushpop_debug_triggers/pushpop_debug_triggers.c

```diff
@@ -293,7 +295,13 @@ static void  test_pop_debug_trigger(void){
 }
 
 int main(int argc, char **argv){
-  let_dmode_setup_triggers();
+  let_dmode_setup_triggers(2);
+  test_push_debug_trigger();
+  test_pop_debug_trigger();
+  let_dmode_setup_triggers(1); // trigger at first address
```

Comment:
Should this be index 0?

Reply:
Visual inspection of the waveform shows that index 1 is correct.

Reply:
I'm not sure, but it lines up in simulation. We could probably double check this.